modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1000,6 +995,34 @@ private ServiceContext _createServiceContext(
 		return serviceContext;
 	}
 
+	private void _disassociateRelatedModel(
+			DTOConverterContext dtoConverterContext,
+			ObjectDefinition objectDefinition,
+			ObjectRelationship objectRelationship, long primaryKey,
+			ObjectDefinition relatedObjectDefinition)
+		throws Exception {
+
+		if (!relatedObjectDefinition.isUnmodifiableSystemObject()) {
+			ObjectEntry objectEntry = fetchRelatedManyToOneObjectEntry(
+				dtoConverterContext, objectDefinition, primaryKey,
+				objectRelationship.getName());
+
+			if (objectEntry != null) {
+				ObjectRelatedModelsProvider objectRelatedModelsProvider =
+					_objectRelatedModelsProviderRegistry.
+						getObjectRelatedModelsProvider(
+							objectDefinition.getClassName(),
+							objectDefinition.getCompanyId(),
+							objectRelationship.getType());
+
+				objectRelatedModelsProvider.disassociateRelatedModels(
+					dtoConverterContext.getUserId(),
+					objectRelationship.getObjectRelationshipId(),
+					objectEntry.getId(), primaryKey);
+			}
+		}
+	}
+
```

Comment:
In the method `dissasociateRelatedModels` in master I see it seems we were supporting many to one relationship. I think we should keep managing in in the same method